main.go

```diff
@@ -135,7 +139,7 @@ func mainerr() error {
 
 	// cache path discovery
 	{
-		gopath = os.Getenv("GOPATH")
+		gopath = goenv.GOPATH
```

Comment:
We effectively had a bug here before. We were only checking the environment values of `GOPATH`, not the effective value (`$(go env GOPATH)`).